try
  io = require 'io.so'
catch error
  print error.stack
